lib/util/pdf.js

```diff
@@ -37,20 +37,24 @@ exports.parse = async function parse (docOptions, callbacks) {
     const scale = 1.0
     const viewport = page.getViewport({ scale })
     const textContent = await page.getTextContent()
-    const textItems = textContent.items.map(item => {
+    const textItems = textContent.items.map((item, index) => {
       const tx = pdfjs.Util.transform(
         viewport.transform,
         item.transform
       )
 
+      let text = item.str
+      if ((index === textContent.items.length - 1) && isNumber(item.str)) {
```

Comment:
Could you explain what is happening here?

Reply:
@jenlky It is incorrect to assume that the page number is always the last element of the textContent. 

The page number may be found on any of the four corners of a PDF document, and sometimes the document may have footer texts, "confidentiality/disclaimer text", dates, and other artefacts.

Reply:
@LoneRifle yes, I made the assumption that page number is the last element based on ExamplePdf. I have included that in the comments. Please take a look.

Reply:
I think that assumption is faulty, at best, and you should not be using the example as the sole basis for your business logic decisions. Further, I expected this to be implemented as a _filter_ on `textContent.items`, which would have allowed us to avoid creating a spurious TextItem containing an empty string

Reply:
Noted. However, I would like to clarify that this was meant for a technical assessment. It's not for any business. I'm simply looking to contribute to open source projects for Hacktober fest. It would be great if you can shed some light on what pdf format this repo was meant to handle.